src/target/riscv/riscv.h

```diff
@@ -236,6 +230,9 @@ struct riscv_info {
 	int (*dmi_read)(struct target *target, uint32_t *value, uint32_t address);
 	int (*dmi_write)(struct target *target, uint32_t address, uint32_t value);
 
+	bool (*get_impebreak)(const struct target *target);
```

Comment:
Detail, if you like:

This callback + the actual functions could be called `has_impebreak` to make it even clearer that they provide a bool value.

```suggestion
	bool (*has_impebreak)(const struct target *target);
```

Reply:
Thanks for the offer, but I will leave previous name to keep it consistent with `get_progbufsize`

Reply:
Ok, let's leave the name as is.